docs/gettsim_objects/input_variables.md

```diff
@@ -110,6 +110,15 @@ Identifier of person that claims Kindergeld for the particular child
 
 Type: int
 
+(wohngeld_kinderzuschl_statt_arbeitsl_geld_2)=
+
+## \`wohngeld_kinderzuschl_statt_arbeitsl_geld_2\`\`
+
+Individual may have claim on Wohngeld and Kinderzuschlag instead of Arbeitslosengeld II
```

Comment:
Maybe

```text
(beantragt_wohngeld_kinderzuschl_statt_arbeitsl_geld_2)=

## \`beantragt_wohngeld_kinderzuschl_statt_arbeitsl_geld_2\`\`

Individual applies for Wohngeld or Kinderzuschlag instead of Arbeitslosengeld II.

Type: bool
```
?

Reply:
But not super-important given overall strategy, maybe add a `temp_` in the name.

Reply:
Ah, now I get it. 

In that case, the Anspruchshöhe for all transfers will be 0. However, if your income is small and that of your children is high, you might end up in a situation where your children form a wthh, and you have your own BG. Result is: Children get Wohngeld, you get nothing (as opposed to everyone in the same BG and the whole FG gets nothing).

Then, `beantragt_wohngeld_kinderzuschl_statt_arbeitsl_geld_2` for you is False, for your children it is True.

Reply:
That was just an example for being ineligible, period. The problem I have with the name that it only makes sense for people eligible for at least one of the programmes.

Reply:
But again, just don't worry much.